app/assets/javascripts/components/course_creator/clone_course.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const CloneCourse = ({ cloneClasss, chooseNewCourseAction, showCloneChooserAction, createLabel, cloneLabel }) => {
```

Comment:
Maybe call this NewOrClone, or something else that will make it easy for us to remember what it's for?

Reply:
It is for the screen which is displayed asking the user to either create new course or clone existing course when the create course button is clicked.

Reply:
Right. So I think 'CloneCourse' may be confusing, as I might assume it's a component focused on choosing a course to clone, or filling out the details to finalize cloning a course, rather than choosing whether to clone a course or start from scratch.